src/Components/WebAssembly/WebAssembly.Authentication/src/RemoteAuthenticatorViewCore.cs

```diff
@@ -138,7 +128,7 @@ protected override void BuildRenderTree(RenderTreeBuilder builder)
                 builder.AddContent(0, CompletingLoggingIn);
                 break;
             case RemoteAuthenticationActions.LogInFailed:
-                builder.AddContent(0, LogInFailed(_message));
+                builder.AddContent(0, LogInFailed(Navigation.HistoryEntryState));
```

Comment:
Could you explain more about what's going on here? It looks as if we're using `Navigation.HistoryEntryState` as part of the user-facing UI now. Is that really where the error message string now lives?

Reply:
It is being passed as part of the state instead of in the query string.

Reply:
> Could you explain more about what's going on here? It looks as if we're using Navigation.HistoryEntryState as part of the user-facing UI now. Is that really where the error message string now lives?

Is it relevant where the message comes from? From the consuming side, you get the message passed down to you as a parameter, consumers don't know if it comes on the query string or on the history entry state.

Reply:
OK, it's fine. Can't remember what I was concerned about here.